pkg/controller/kms/alias/controller.go

```diff
@@ -14,8 +14,6 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Code generated by ack-generate. DO NOT EDIT.
-
```

Comment:
Could be worth a note somewhere to explain why this resource looks like generated code, but "isn't".

Reply:
Added a comment to all those files pointing to description of this PR for details.